Libero_Projects/import/components/IMC_DGC2/README.md

```diff
@@ -0,0 +1,138 @@
+## Mi-V Extended Subsystem Design Guide Configuration 2: I2C Write & Boot
+This folder contains Tcl scripts that build Libero SoC v2021.3 MIV_ESS DGC2 design project for the Future Avalanche Board. The script is executed in Libero SoC to generate the sample design. 
+
+#### PF_Avalanche_MIV_RV32_BaseDesign (no ES equivalent)
+
+| Config  | Description|
```

Comment:
Remove the "Features" section at the bottom of the readme add the list of features as a third column to this table.

Reply:
I ran out of time to make all of the changes, I'll make a note of this one. I might get a chance to include this in while waiting for PR review.